src/components/Router/StepComponentMap.js

```diff
@@ -0,0 +1,58 @@
+import { h, Component } from 'preact'
+import { events } from 'onfido-sdk-core'
+
+import Welcome from '../Welcome'
+import Select from '../Select'
+import {FrontDocumentCapture, BackDocumentCapture, FaceCapture} from '../Capture'
+import Complete from '../Complete'
+
+export const createComponentList = (stepOptions, componentOptions) => {
+  const formattedStepOptions = shallowFlatten(stepOptions.map(formatStep))
+  const stepCount = formattedStepOptions.length
+  const mapSteps = (step, index) => {
+    const options = {step, componentOptions}
```

Comment:
Minor: there are so many option objects already, I would just pass this parameters the classic way tbh